modules/prebidServerBidAdapter.js

```diff
@@ -695,7 +695,10 @@ export function PrebidServer() {
 
     ajax(
       _s2sConfig.endpoint,
-      response => handleResponse(response, requestedBidders, bidRequests, addBidResponse, done),
+      {
+        success: response => handleResponse(response, requestedBidders, bidRequests, addBidResponse, done),
+        error: () => { done(); }
```

Comment:
Should this error be logged somewhere, so it shows up in debug mode?

If not: `error: done` should work here. No need to wrap the function in another function.